src/components/compliance/compliance.css

```
@@ -1,212 +1,212 @@
-	}
-
-	& .compliance .tier.tier--2 {
-		/* Appearance */
-		background: #3a87cf;
-	}
-
-	& .compliance .tier.tier--3 {
-		/* Appearance */
-		background: #81009a;
-	}
-
-	& .beginning-headline {
-		/* Layout */
-		margin-block-end: 4.5rem;
-	}
-
-	& .ending-headline {
-		/* Layout */
-		margin-block-end: 3.5rem;
-	}
-
-	& .tier {
-		/* Layout */
-		block-size: 100%;
-		margin-inline-end: 1rem;
-		padding: .125rem .75rem;
-
-		/* Text */
-		font-size: .75em;
-		font-style: normal;
-		font-weight: 700;
-		letter-spacing: .005em;
-		line-height: 4--step;
-		white-space: nowrap;
-
-		/* Appearance */
-		border-radius: 3px;
-		color: #ffffff;
-	}
-
-	& .rule-text[data-status="under-review"] .rule-status {
-		/* Appearance */
-		color: #c8102e;
-	}
-
-	& .rule-text[data-status="updated"] .rule-status {
-		/* Appearance */
-		color: #786dd3;
-	}
-
-	& .rule-text[data-status="deprecated"] .rule-status {
-		/* Appearance */
-		color: #51555b;
-	}
-
-	& .rule {
-		display: flex;
-
-		/* Layout */
-		margin-block-end: 1.25rem;
-	}
-
-	& .ruleset-link {
-		display: inline-flex;
-
-		/* Layout */
-		align-items: center;
-		margin-inline-start: 1rem;
-
-		/* Text */
-		font-size: .875rem;
-		font-weight: 400;
-		line-height: 1rem;
-	}
-
-	& .subname {
-		/* Text */
-		font-size: 5--step;
-		font-weight: 700;
-	}
-
-	& .rules-name {
-		display: flex;
-
-		/* Layout */
-		margin-block: 10--step 5--step;
-
-		/* Text */
-		font-size: 5--step;
-		font-weight: 700;
-	}
-
-	& .ruleset-name {
-		/* Layout */
-		margin-inline-start: 1rem;
-	}
-
-	& .ruleset-general {
-		display: block;
-
-		/* Layout */
-		margin-block-start: 1.5rem;
-	}
-
-	& :where(li)::before {
-		/* Generated */
-		content: "";
-	}
+    /* reference */
+    --BrightBlue600Color: hsl(209 61% 52%);
+    --Pink600Color: hsl(290 100% 30%);
+    --Teal600Color: hsl(181 100% 32%);
+
+    & dl {
+        display: flex;
+
+        /* layout */
+        align-items: flex-start;
+        gap: 4--step;
+    }
+
+    & .tier {
+        /* layout */
+        padding: 0.125rem 0.75rem;
+
+        /* typography */
+        font-weight: 700;
+        font-size: 3--step;
+        line-height: calc(16 / 12);
+        text-align: center;
+        white-space: nowrap;
+
+        /* decoration */
+        background-color: var(--color);
+        border-radius: 3--rpx;
+        color: var(--PrimaryColor);
+
+        &.t1 {
+            --color: var(--Teal600Color);
+        }
+
+        &.t2 {
+            --color: var(--BrightBlue600Color);
+        }
+
+        &.t3 {
+            --color: var(--Pink600Color);
+        }
+    }
+
+    & .p {
+        line-height: 1.5;
+        padding-top: 0.4rem;
+        margin-bottom: 0.8rem;
+      }
+
+    & .status {
+        white-space: nowrap;
+        height: 100%;
+        border-radius: 3px;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 0.75em;
+        letter-spacing: 0.005em;
+        line-height: 4--step;
+        color: #fff;
```

May we use the token here? I can also add these later in a separate PR.

```suggestion
        color: var(--PrimaryColor);
```